flixel/system/frontEnds/HTML5FrontEnd.hx

```diff
@@ -59,6 +61,49 @@ class HTML5FrontEnd
 	{
 		return Browser.window.innerHeight;
 	}
+	
+	private inline function get_platform():FlxPlatform
+	{
+		
+		if (Browser.navigator.userAgent.indexOf("Win") > -1)
```

Comment:
Should probably wrap `Browser.navigator.userAgent.indexOf()` in a helper function, it's used quite often here.